resources/session02/homework/http_server.py

```diff
@@ -35,7 +42,37 @@ def parse_request(request):
 
 def resolve_uri(uri):
     """This method should return appropriate content and a mime type"""
-    return b"still broken", b"text/plain"
+    contents = b''
+    mimetype = b''
+    directory = './webroot'
+
+    if isinstance(uri, bytes):
+        directory += str(uri.decode('utf8'))
+    else:
+        directory += str(uri)
```

Comment:
In this we have the implicit assumption that URI will always be an absolute path.  What if it is not?